.travis.yml

```diff
@@ -2,6 +2,9 @@ language: erlang
 
 otp_release:
   - 18.0
+  - 18.1
+  - 18.2.1
```

Comment:
I just took everything mentioned by @eksperimental and the erlang download page (same releases) - happy to adapt the PR to run latest patch level for all :)